agent/replacecall/src/main/java/randoop/grt_mock/javax/mail/Service.java

```diff
@@ -0,0 +1,27 @@
+package randoop.grt_mock.javax.mail;
```

Comment:
I think these can be in `randoop.mock` rather than a separate `randoop.grt_mock`.  Other users may find these useful.